packages/@okta/vuepress-site/use_cases/inline_hooks/registration_hook/registration_hook/index.md

```diff
@@ -0,0 +1,244 @@
+
+The action that Okta is currently set to take, regarding whether to allow this registration attempt.
+
+There are two possible values:
+
+- `ALLOW` indicates that the registration attempt will be allowed to proceed
+- `DENY` indicates that the registration attempt will be terminated (no user will be created in Okta)
+
+The action is `ALLOW` by default (in practice, `DENY` will never be sent to your external service).
+
+By means of the `com.okta.action.update` [command](#supported-commands) in your response, you can change the action that Okta will take.
+
+## Objects in Response You Send
+
+For the Registration Inline Hook, the `commands` and `error` objects that you can return in the JSON payload of your response are defined as follows:
+
+### commands
+
+The `commands` object lets you invoke commands to modify or add values to the attributes in the Okta user profile that will be created for this user, as well as to control whether or not the registration attempt is allowed to proceed.
+
+This object is an array, allowing you to send multiple commands in your response. Each array element requires a `type` property and a `value` property. The `type` property is where you specify which of the supported commands you wish to execute, and `value` is where you supply parameters for that command.
+
+| Property | Description                                           | Data Type       |
+|----------|-------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands). | String          |
+| value    | Operand to pass to the command.                       | [value](#value) |
+
+For example commands, see the [value](#value) section below.
+
+#### Supported Commands
+
+The following commands are supported for the Registration Inline Hook type:
+
+| Command                      | Description                                                       |
+|------------------------------|-------------------------------------------------------------------|
+| com.okta.user.profile.update | Set or modify an attribute in the Okta user profile that will be created for this user.                 |
+| com.okta.action.update       | Allow or deny the user's registration. |
+
+To set attributes in the user's Okta profile, supply a type property set to `com.okta.user.profile.update`, together with a `value` property set to a list of key-value pairs corresponding to Okta user profile attributes you want to set. The attributes must already exist in your user profile schema.
+
+To explicitly allow or deny registration to the user, supply a type property set to `com.okta.action.update`, together with a value property set to `{"registration": "ALLOW"}` or `{"registration": "DENY"`}. The default is to allow registration.
+
+Commands are applied in the order in which they appear in the array. Within a single command, attributes are updated in the order in which they appear in the `value` object.
```

Comment:
This was probably my own oversight: "Within a single **com.okta.user.profile.update** command, attributes..."

Reply:
Added the name of the command (you probably had it originally and I dropped it).